qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -551,13 +551,22 @@ def on_mode_left(self, mode):
                 return
             widget.setFocus()
 
+    def load_prepared_history(self, idx):
+        tab = self.widget(idx)
+        if len(tab.history_prepared) > 0:
+            tab.history.load_items(tab.history_prepared)
+            tab.history_prepared = []
+        #end if
+    #end def load_prepared_history
```

Comment:
Please remove those comments (not only here, in other locations too) - they might make sense if you come from Ruby or so, but are not something commonly done in Python.